### DISCUSSION TITLE
How to blacklist paid content from unsubscribed accounts?

### DISCUSSION THREAD

**Discussion by CheeseburgerWithFries** [UltimaHoarder/UltimaScraper#1406] [Q&A]
Once I unsubscribe from an account, I move their folder out of the .site/Onlyfans directory into a different folder so I can organize it. However, any paid content will download again, taking up space on my hard drive. How do I set things up so that the scraper doesn't redownload _all_ paid content?

Things I've already tried in .settings/config.json:
Setting `"scrape_paid_content": true` to `false` - This disables ALL paid content, including new paid content I want downloaded.
Creating a list on the site with unsubscribed accounts and setting `"blacklist_name": "Nah"` to point there - Doesn't work, seems like paid content scrape is separate from the username scrape where the blacklist applies.
Putting the username under `"ignored_keywords": ["username"]` - Didn't work, no idea why.

How do I fix this? It wasn't an issue until a couple months ago.

**Comment by UltimaHoarder** [comment]
Change the metadata directory to something static. Like ".Metadata"

**Reply by CheeseburgerWithFries** [reply]
That didn't work. If I change it to Username/.Metadata, the scraper will just create a new Metadata folder without the `.` and download the files anyway.

**Reply by UltimaHoarder** [reply]
Remove the username part.

**Reply by CheeseburgerWithFries** [reply]
That's where the Metadata folders are though. There is no Metadata folder in .sites/Onlyfans, they're located in .sites/Onlyfans/Username/

**Reply by UltimaHoarder** [reply]
You have to change it. Read the readme.

**Reply by CheeseburgerWithFries** [reply]
No joy. I tried changing the config.json to both `"metadata_directory_format": "{site_name}\\Metadata"` and `"{site_name}\\.Metadata"` per the readme and your instructions, and the scrape time went from about 6 minutes to upwards of 30 minutes and threw errors randomly on different profiles each time. Couldn't get through the scrape to verify whether the static folder idea worked or not.
